src/main/java/cloud/fogbow/ras/api/http/response/quotas/allocation/ResourceAllocation.java

```diff
@@ -0,0 +1,117 @@
+package cloud.fogbow.ras.api.http.response.quotas.allocation;
+
+import javax.persistence.Column;
```

Comment:
i'm unsure, but shouldn't we use the spring api?

Reply:
I used the same imports used in the **ComputeAllocation** class, I believe the Spring JPA implementation also works with these annotations.